database.py

```diff
@@ -93,6 +93,14 @@ def get_extra(self):
         # Get all the Extra Pokemon images available.
         return self.__get_region(None)
 
+    def get_light(self):
```

Comment:
If you want to combine it into one "get_light_or_dark" function, you still need an `elif arg in ("light","dark") and escape_code` clause before that to set the wallpaper. As for choosing separate functions for light and dark, I just did whatever matched the existing functions and arguments most.